### ISSUE TITLE
Fingerprint security key option not found when adding the device to an existing account

### ISSUE THREAD

**Issue by neokree** [dfinity/internet-identity#1086]
Hello, 

I recently bought a new phone, and I was trying to add it to my existing internet identity, only to find out that there isn't an option for it. 
![Screenshot_20221211-142908](https://user-images.githubusercontent.com/9344184/206913212-3e70370c-cd4f-4123-9862-9061d9c34623.png)

To be sure, with the same browser (Brave) I tried to create a new Identity, and this time the fingerprint option was available. 

Is there a reason why you are not supporting the use of a fingerprint option when adding the device to an existing account? 

Tested with a Google Pixel 6 with Android 13 installed

**Comment by nmattia** [comment]
Hi! apologies for the delay, was off for a few weeks.

That's really surprising. I can actually reproduce this... but not with every anchor, which is really odd. Let me investigate a bit and come back to you; thanks for the report!

**Comment by neokree** [comment]
Hi @nmattia I also found out that the biometric authentication is available if you first recover your anchor using the seed on the new device, and after logging in you add your new device.

**Comment by neokree** [comment]
Maybe it's something that happens only the first time you try to connect with a new device to an existing anchor? 
I can reproduce it every time by trying to login to a random anchor

**Comment by nmattia** [comment]
Ok, so we do exclude some authenticators when creating credentials for a new device:

https://github.com/dfinity/internet-identity/blob/b66fe925fb0a337b09aaaa5beaf1a60994b19f14/src/frontend/src/flows/addDevice/welcomeView/registerTentativeDevice.ts#L142

I'm not 100% sure why it's there and what it does; @frederikrothenberger can you clarify why it's needed and what exactly it does?

**Comment by frederikrothenberger** [comment]
This excludes already registered devices. I.e. it prevents creating a new credential if an existing one is already available on the same WebAuthn `Authenticator`. So you cannot register TouchId twice using the same MacBook, but you can register TouchID and a YubiKey that is attached to the same MacBook because the YubiKey is considered a different `Authenticator`.

**Comment by nmattia** [comment]
@frederikrothenberger what happens if you have two anchors, 10000 and 10001, and 10000 has TouchID and you want to add TouchID to 10001? Is TouchID on 10000 considered registering TouchID twice?

**Comment by frederikrothenberger** [comment]
@nmattia: No, the excluded credential id's are the ones of the anchor you're trying to add a device to. So, when adding something to anchor 10000 you don't exclude any credentials used on other anchors.

**Comment by aterga** [comment]
Closing — the device management flow has been significantly reworked since this was filed. The WebAuthn `excludeCredentials` handling and the UI for adding devices have both been redesigned.
